backend/apps/volontulo/tests/views/offers/test_offers_list.py

```diff
@@ -1,63 +1,91 @@
-# -*- coding: utf-8 -*-
-
 """
 .. module:: test_offers_list
 """
 
 from django.test import Client
 from django.test import TestCase
 
-from apps.volontulo.tests.views.offers.commons import TestOffersCommons
+from apps.volontulo.factories import OfferFactory
+from apps.volontulo.factories import OrganizationFactory
+from apps.volontulo.factories import UserFactory
 
 
-class TestOffersList(TestOffersCommons, TestCase):
+class TestOffersList(TestCase):
     """Class responsible for testing offers' list."""
 
     def setUp(self):
         """Set up each test."""
         self.client = Client()
 
-    def _test_offers_list_for_standard_user(self):
-        """Test offers' list for standard user.
-
-        List of offers is available for standard users and shows only ACTIVE
-        offers.
-        Test are common for anonymous user, volunteer and organization.
-        """
-        response = self.client.get('/o/offers')
+    def _test_offers_list(self, response):
+        """Test offers' list."""
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'offers/offers_list.html')
         self.assertIn('offers', response.context)
-        self.assertEqual(len(response.context['offers']), 1)
 
     def test_offer_list_for_anonymous_user(self):
         """Test offers' list for anonymus user."""
-        return self._test_offers_list_for_standard_user()
+        OfferFactory.create_batch(
+            31,
+            offer_status='published',
+            finished_at=None,
+            recruitment_end_date=None,
+        )
+        OfferFactory.create_batch(48, offer_status='unpublished')
+        OfferFactory.create_batch(52, offer_status='rejected')
+
+        response = self.client.get('/o/offers')
```

Comment:
`/o/offers/` or `/o/offers`?

Reply:
AFAIR there was redirect without slash at the end, but here it shouldn't be a problem.

Reply:
That's old, Django-based test. As soon as we move exclusivelly to Angular and API endpoints it will be removed.

Also it's thre without tailing `/`: https://github.com/CodeForPoznan/volontulo/blob/master/backend/apps/volontulo/urls.py#L83